frontend/src/e2e-playwright/specs/4_finance/value-decisions.spec.ts

```diff
@@ -119,6 +119,13 @@ describe('Value decisions', () => {
     await valueDecisionsPage.navigateBackFromDetails()
   })
 
+  test('Send value decision from details page', async () => {
+    await valueDecisionsPage.openFirstValueDecision()
+    await valueDecisionsPage.sendValueDecision()
+    await valueDecisionsPage.navigateBackFromDetails()
+    await waitUntilEqual(() => valueDecisionsPage.getValueDecisionCount(), 1)
```

Comment:
Am I missing something or shouldn't you change update the filter to show sent decisions?

Reply:
Well this is the count of draft invoices left (was originally 1), but yes, it is better to check that the decision was actually sent